webpack.config.js

```diff
@@ -46,7 +46,13 @@ const baseConfig = mode => ({
       resourceRegExp: /@blueprintjs\/(core|icons)/, // ignore optional UI framework dependencies
     }),
   ],
-  resolve: { extensions: ['.js'] },
+  resolve: {
+    alias: {
+      // needs shared global state for context to work
+      'react-dnd': path.resolve(path.join(__dirname, 'node_modules', 'react-dnd')),
```

Comment:
I don't know how acceptable this is, but it seems like it works well enough for now?

Reply:
Yeah it will be fine for this UMD build. However I think adopters who build their own package may need to do something similar

Reply:
And they'd probably want to anyway, for exactly the same reasons